src/components/data/DataTable.js

```diff
@@ -346,6 +372,12 @@ const DataTable = props => {
               }
             }
 
+            const filterBreadCrumb = h(div, { style: { display: 'flex', overflow: 'hidden' } }, [h(TextCell, { style: { fontWeight: 400, ...Style.noWrapEllipsis } }, [`filtered by: ${filterColTerm}`]), h(Clickable, {
+              'aria-label': 'Clear filter',
+              tooltip: 'Clear filter',
+              style: { alignSelf: 'flex-start', marginLeft: 'auto' },
+              onClick: () => setColumnFilter('')
+            }, [icon('times-circle', { color: colors.light(8), size: 16 })])])
```

Comment:
real minor nit: when the `Style.noWrapEllipsis` does NOT kick in - when the filter value is short - the "x" icon is awfully close to the value:
![Screenshot 3-14-2023 at 05 34 PM](https://user-images.githubusercontent.com/6041577/225143373-3d8b3ce8-6001-4706-9b42-1a028b0d36a3.png)

Reply:
Also … when clicking the "x" to remove the filter … it changes the sort for that column as well :(

Reply:
I cannot tell you how long I've spent trying to put space between the 'x' and the term.  I don't know why nothing I try works.

Reply:
eh, then I say don't worry about it, let's get this PR in and potentially/maybe work on that spacing again later.